docs/src/time_integration.md

```diff
@@ -85,7 +85,7 @@ First, you need to load the necessary packages:
 
 ```@example PERK-example-1
 using Convex, ECOS, NLsolve
-using OrdinaryDiffEq
+using OrdinaryDiffEqLowStorageRK
```

Comment:
```suggestion
```
This should not be necessary anymore as this was only added for the `CallbackSet`.